archaius-core/src/main/java/com/netflix/config/ConfigurationManager.java

```diff
@@ -241,8 +242,17 @@ public static void loadPropertiesFromResources(String path)
             URL url = resources.nextElement();
             Properties props = new Properties();
             InputStream fin = url.openStream();
-            props.load(fin);
-            fin.close();
+            InputStreamReader reader = new InputStreamReader(fin, "UTF-8");
```

Comment:
This code path is seldom used. But I'm fine with fixing it here as well. Please consolidate the functionality here and put it in a function inside ConfigurationUtils class so we don't have duplicate code in two places.